assets/css/components/_segmented-control.scss

```diff
@@ -0,0 +1,26 @@
+.segmented-control {
+  box-shadow: inset 0 -1px $light-gray;
+  display: flex;
+  font-size: $small-font-size;
+  overflow-x: auto;
+  width: 100%;
+
+  @include media($small-screen-up) {
+    @include padding(null ($base-spacing - $small-spacing));
+  }
+
+  a {
+    display: inline-block;
+    padding: $small-spacing;
+    white-space: nowrap;
+
+    &.selected {
+      box-shadow: inset 0 -2px $dark-gray;
```

Comment:
I agree with this, but left it because it will require a good amount of refactoring because our `class_for` helper doesn’t allow us to pass in arbitrary classes to go alongside the `selected` class. So this nesting is needed…